org.eclipse.xtext.builder.standalone.tests/src-gen/org/eclipse/xtext/builder/tests/parser/antlr/internal/InternalBuilderTestLanguageLexer.java

```diff
@@ -862,78 +862,75 @@ public void mTokens() throws RecognitionException {
         "\2\uffff\1\2\1\3\3\uffff\1\10\1\11\1\uffff\1\12\1\13\3\uffff\1\17\1\20\1\uffff\1\12\1\2\1\3\4\uffff\1\10\1\11\1\13\1\14\1\15\1\16\1\17\25\uffff\1\4\1\5\10\uffff\1\1\1\7\1\uffff\1\6";
     static final String DFA12_specialS =
         "\1\2\13\uffff\1\0\1\1\65\uffff}>";
-    static final String[] DFA12_transitionS = DFA12_transitionS_.DFA12_transitionS;
-    private static final class DFA12_transitionS_ {
```

Comment:
Did you generate with an old version?

Reply:
run on master, but may be executed with older version?